tools/ctl/ide/ui.go

```diff
@@ -207,6 +218,65 @@ func (e *explorer) showDetails(ctx context.Context, example rundex.Rebuild) {
 	showModal(e.app, e.container, details, modalOpts{Margin: 10})
 }
 
+func downloadLogs(ctx context.Context, localAssets rebuild.AssetStore, example rundex.Rebuild) error {
```

Comment:
ugh. this makes me want to copy the logs to the new debug bucket. I get that it's potentially a waste of resources but this access pattern just doesn't bring me joy (especially given the additional CLI flag). I'm fine with it for now but it might be nice to copy to the debug storage after the GCB run to eliminate this complexity. Maybe add a TODO?

Reply:
Added a TODO to maybe copy the logs to debug bucket.